### DISCUSSION TITLE
关于倒计时countDown组件

### DISCUSSION THREAD

**Discussion by Bryant24s** [TaleLin/lin-ui#1407] [求助]
如下图，时间数字大小样式是可以修改的。但是，如何修改旁边文字“天-时-分-秒”大小。如何修改冒号“：”大小样式
![1625487326(1)](https://user-images.githubusercontent.com/14868600/124470426-2cd39f80-ddce-11eb-8bad-e5683ffacf0f.png)

**Comment by aQingYun** [comment]
可以使用l-class修改下font-size试下

**Reply by Bryant24s** [reply]
ok，好的，也只能这样啦
